modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/test/java/com/liferay/dynamic/data/mapping/internal/util/DDMIndexerImplTest.java

```diff
@@ -475,6 +477,42 @@ private void _testFormWithRepeatableField(String indexType) {
 			StringPool.BLANK);
 	}
 
+	private void _testFormWithRepeatableRichTextField() {
+		Document document = _createDocument();
+
+		_ddmIndexer = _createDDMIndexer(true);
+
+		DDMForm ddmForm = DDMStructureTestUtil.getSampleDDMForm(
+			_FIELD_NAME, "string", "text", true,
+			DDMFormFieldTypeConstants.RICH_TEXT,
+			new Locale[] {LocaleUtil.getSiteDefault()},
+			LocaleUtil.getSiteDefault());
+
+		List<DDMFormField> ddmFormFieldList = ddmForm.getDDMFormFields();
+
+		DDMFormField ddmFormField = ddmFormFieldList.get(0);
+
+		DDMStructure ddmStructure = _createDDMStructure(ddmForm);
+
+		_setUpHtmlParser();
```

Comment:
Hi @carolmariaabb !

I tried to call this code on setUp() method as you requested but nothing happened, instead the test was failing and when debugging it I saw that the htmlParser was null; this was the only way I managed to fix it.